pkg/contracts/src/OsmoticPool.sol

```diff
@@ -192,6 +152,50 @@ contract OsmoticPool is Initializable, OwnableUpgradeable, OsmoticFormula {
         supportProjects(_projectSupports);
     }
 
+    /* *************************************************************************************************************************************/
+    /* ** Project Activation Function                                                                                                    ***/
+    /* *************************************************************************************************************************************/
+
+    function activateProject(uint256 _projectId) public {
+        if (!IProjectList(projectList).projectExists(_projectId)) {
+            revert ProjectNotInList(_projectId);
+        }
+
+        uint256 projectSupport = getProjectSupport(_projectId);
+
+        if (projectSupport == 0) {
```

Comment:
I initially considered a scenario where, with the maximum number of active projects, a caller wants to activate a new one. This would revert to `ProjectNeedsMoreStake` since its support is zero. Reverting sooner could save the caller some gas.

However, upon further reflection, I believe it's better to enforce this constraint on the UI side rather than within the contracts themselves, as there is no real contract risk associated with executing this call under these conditions.

Let's remove this check and any related unit tests.